vault/expiration_test.go

```diff
@@ -2943,6 +2956,14 @@ func TestExpiration_StopClearsIrrevocableCache(t *testing.T) {
 	if _, ok := exp.irrevocable.Load(leaseID); ok {
 		t.Error("expiration manager irrevocable cache should be cleared on stop")
 	}
+
+	exp.pendingLock.Lock()
```

Comment:
all of these reads can just be under `RLock()` instead of `Lock()` (and then `RUnlock()`)

Reply:
ah yeah good catch! df2931a